client/sass/application.scss

```diff
@@ -1,3 +1,63 @@
 $govuk-global-styles: true;
 
 @import "node_modules/govuk-frontend/govuk/all";
+
```

Comment:
Assume we have added some custom styling to the nav bar as there is nothing suitable in the `govuk-frontend`?

Reply:
You got it! From speaking with our designer the design for this came from the design system itself, but on their docs page 😁 

![Screenshot 2023-10-25 at 16 30 34](https://github.com/DEFRA/water-abstraction-system/assets/1789650/8bee2ddf-e9ca-4f47-8799-43de0470373d)

It's not actually a component of the frontend. He'd like to update it to match what the design system has now but we'd need to wait until all pages were migrated to system.